--- mpt-status.h.orig	2007-01-16 15:07:36.000000000 +0100
+++ mpt-status.h	2007-01-16 22:43:15.000000000 +0100
@@ -15,13 +15,13 @@
 #ifndef __kernel
 #define __kernel
 #endif
-#include "pci.h"	// config.h and header.h from pciutils package
-#include "lsi/mpi_type.h"
-#include "lsi/mpi.h"
-#include "lsi/mpi_ioc.h"
-#include "lsi/mpi_cnfg.h"
-#include "lsi/mpi_raid.h"
-#include "mptctl.h"
+#include <pci/pci.h>	// config.h and header.h from pciutils package
+#include "debian/includes/mpi_type.h"
+#include "debian/includes/mpi.h"
+#include "debian/includes/mpi_ioc.h"
+#include "debian/includes/mpi_cnfg.h"
+#include "debian/includes/mpi_raid.h"
+#include "debian/includes/mptctl.h"
 //#include "mptbase.h"
 #endif // SANITIZED_KERNEL_HEADERS
 
